content/reference/stacks.md

```diff
@@ -0,0 +1,277 @@
+---
+title: "Stacks"
+icon: "fa fa-layer-group"
+description: "Stacks are a way to express the complete infrastructure needed for an environment using a standard YAML configuration"
+weight: 1
+tags:
+- yaml
+- atmos
+- configuration
+- concept
+---
+
+# Background
+
+Stacks are a central SweetOps abstraction layer that are used to instantiate components. They're a set of YAML files [that follow a standard schema](https://github.com/cloudposse/atmos/blob/master/docs/schema/stack-config-schema.json) to enable a **fully declarative description of your various environments**. This empowers you with the ability to separate your infrastructure's environment configuration settings from the business logic behind it (provided via components).
+
+SweetOps utilizes YAML configuration for stacks because it's an easy to work with format that is nicely portable across multiple tools. The stack yaml format is natively supported today via [atmos](https://github.com/cloudposse/atmos), [the terraform-yaml-stack-config module](https://github.com/cloudposse/terraform-yaml-stack-config), and [Spacelift](https://spacelift.io) via [the terraform-spacelift-cloud-infrastructure-automation module](https://github.com/cloudposse/terraform-spacelift-cloud-infrastructure-automation).
+
+# Conventions
+
+We have a number of important conventions around stacks that are worth noting.
+
+## Stack Files
+
+Stack files can be very numerous in large cloud environments (think many dozens to hundreds of stack files). To enable proper organization of stack files, SweetOps recommends the following:
+
+* All stacks should be stored in a `stacks/` folder at the root of your infrastructure repository.
+* Name individual environment stacks following the pattern of `$environment-$stage.yaml`
+  * For example, `$environment` might be `ue2` (for `us-east-2`) and `$stage` might be `prod` which would result in `stacks/ue2-prod.yaml`
+* For any **global** resources, such as Account Settings, IAM roles and policies, DNS records, or similar, the `environment` for the stack should be `gbl`.
+  * For example, to deploy the `iam-delegated-roles` component (where all resources are global and not associated with an AWS region) to your production account, you should utilize a `stacks/gbl-prod.yaml` stack file.
+
+## Shared Stack Configuration
+
+When you have configuration that you want to share across various stacks, use catalogs. Catalogs are the SweetOps term for shared, reusable configuration.
+
+By convention, all shared configuration for stacks is put in the `stacks/catalogs/` folder, which can then be used in the root `stacks/` stack files via `import`.
+
+There are a few types of shared configuration which are worth noting:
+
+* **Global Catalogs**: For any configuration to share across **all** stacks.
+  * For example, you create a `stacks/catalogs/globals.yaml` file and utilize `import` wherever you need that catalog.
```

Comment:
It appears we (cloudposse) are inconsistent with calling directory `catalogs` or `catalog`, but I looked at a few of our modules, and it appears we use `catalog`. Also, it makes sense since we have a single catalog with lots of configurations.

Reply:
Good catch 👍